roofit/histfactory/test/testHistFactory.cxx

```diff
@@ -711,18 +711,5 @@ INSTANTIATE_TEST_SUITE_P(HistFactory, HFFixtureFit,
                          testing::Combine(testing::Values(MakeModelMode::OverallSyst, MakeModelMode::HistoSyst,
                                                           MakeModelMode::StatSyst, MakeModelMode::ShapeSyst),
                                           testing::Values(false, true), // non-uniform bins or not
-                                          testing::Values(ROOFIT_EVAL_BACKENDS)),
-                         getNameFromInfo);
-
-#if !defined(_MSC_VER) || defined(R__ENABLE_BROKEN_WIN_TESTS) // See https://github.com/vgvassilev/clad/issues/752
-#ifdef TEST_CODEGEN_AD
-// TODO: merge with the previous HFFixtureFix test suite once the codegen AD
-// supports all of HistFactory
-INSTANTIATE_TEST_SUITE_P(HistFactoryCodeGen, HFFixtureFit,
-                         testing::Combine(testing::Values(MakeModelMode::OverallSyst, MakeModelMode::HistoSyst,
-                                                          MakeModelMode::StatSyst, MakeModelMode::ShapeSyst),
-                                          testing::Values(false), // no non-uniform bins
-                                          testing::Values(RooFit::EvalBackend::Codegen())),
+                                          testing::Values(ROOFIT_EVAL_BACKENDS_WITH_CODEGEN)),
```

Comment:
@pcanal, many tests were added by extending the test matrix to also do non-uniform binning fits with the `Codegen()` backend (the `ROOFIT_EVAL_BACKENDS_WITH_CODEGEN` is a macro that conditionally includes the `codegen` backend if ROOT was built with Clad).

I confirmed locally that all HistFactory fitting tests now run with all backends as expected.

Seems that we were also lucky and some Windows issues with Clad in the HistFactory tests was gone, presumably because of the LLVM upgrade.

Reply:
Perfect.  Thanks for the clarification.